CONTRIBUTING.md

```diff
@@ -52,27 +52,42 @@ gem server, please consider checking out
 provide pass-through caching for RubyGems.org, as well as host private
 gems for your organization..**
 
+### Getting the code
+
+Clone the repo: `git clone git@github.com:rubygems/rubygems.org.git`
+Move to the newly cloned repository directory: `cd rubygems.org`
+
+### Setting up the environment
+
+Rubygems.org is a Ruby on Rails application.
+The app depends on Elasticsearch, Memcached, and Postgres.
+Google Chrome is used for tests.
+
+Setup the development environment using one of the approaches below.
+
 #### Environment (Docker)
 
 There is a `docker-compose.yml` file inside the project that easily lets you spin up
-services that the application depends on such as: postgresql, memcached & elasticsearch.
+postgresql, memcached & elasticsearch.
 
-* Install Docker. See instructions at https://docs.docker.com/engine/installation/
-* run `docker-compose up` to start the required services.
+Note: Docker compose does not run the rubygems.org application itself.
+
+* Install Docker. See instructions at https://docs.docker.com/get-docker/
+* run `docker compose up` to start the required services.
 
 Follow the instructions below on how to install Bundler and setup the database.
 
 #### Environment (OS X)
 
-* Use Ruby 3.1.x (`.ruby-version` is present and can be used)
-* Use Rubygems 3.3.x
-* Install bundler: `gem install bundler`
```

Comment:
Moved these instructions to a specific "install ruby and dependencies" section since they were repeated for OS X and Linux and skipped for docker even though all 3 will need virtually identical instructions.